src/main/java/core/basesyntax/util/filewriter/FileWriterImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.util.filewriter;
+
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String resultingReport, String toFileName) {
+        try (java.io.FileWriter fileWriter = new java.io.FileWriter(toFileName)) {
```

Comment:
It's good to see the use of try-with-resources here to ensure the FileWriter is closed properly. However, make sure to avoid using fully qualified class names with packages inline. It's better to import the class at the top of the file.